_posts/atoms/2018-08-30-table.html

```diff
@@ -34,27 +34,26 @@
 
 %h5 Table with DataTable
 
-%table.responsive.table.table-striped.table-bordered#my-datatable
-  %thead
-    %tr
-      %th Fruits
-      %th Origin
-      %th Vendor
-  %tbody
-    %tr
-      %td Oranges
-      %td Bananas
-      %td Kiwis
-    %tr
-      %td Spain
-      %td Brazil
-      %td Germany
-    %tr
-      %td Fruits Are Us
-      %td Fruits & Vegetables Inc.
-      %td SUSE Fruit Company Ltd.
+.obs-dataTable
```

Comment:
This is not needed, please remove it.

Reply:
of course it is! This is what is solving the bug (https://github.com/mkhairi/jquery-datatables/issues/14). Check:

https://github.com/openSUSE/open-build-service/blob/master/src/api/app/assets/stylesheets/webui2/webui2.css.scss#L54

Reply:
Oh, alright, could this be explained in the page.